wiki/FitNesseRoot/HsacExamples/SlimTests/HttpTests/content.txt

```diff
@@ -5,6 +5,7 @@ The tests in this suite show example usage of this fixture:
 >HttpGetTest shows how an example GET requests, specifying parameters in the query string.
 >HttpPost1Test sends a POST request to a SOAP service, also adding an HTTP header.
 The >HttpPost2UsingScenarioTest makes multiple SOAP calls to a service, where scenario parameters are used to change an input variable's value for each call.
+>HttpPostFileTest sends a binary file to a mocked endpoint.
```

Comment:
Here I would just mention it posts a binary file, and in the page describe its to a mocked endpoint